packages/analytics/test/expand.js

```diff
@@ -275,6 +275,43 @@ test('with no path', (done) => {
             done();
         });
 });
+test.only('with no value #1', (done) => {
```

Comment:
Warning: there is still an `only` there!

```suggestion
test('with no value #1', (done) => {
```